templates/password.json

```diff
@@ -4,13 +4,27 @@
     "main": {
       "type": "newsletter",
       "settings": {
-        "heading": "Opening soon",
-        "subheading": "<p>Be the first to know when we launch.</p>",
         "full_width": false
-      }
+      },
+      "blocks": {
+        "heading": {
+          "type": "heading",
+          "settings": {
+            "heading": "Opening soon"
+          }
+        },
+        "subheading": {
```

Comment:
Perhaps we should remain consistent with the block names defined in [`sections/newsletter.liquid`](https://github.com/Shopify/dawn/blob/main/sections/newsletter.liquid#L117).

Reply:
the name of the block is `subheading` that's why I thought it would make sense to keep it this way. [Screenshot](https://screenshot.click/21-06-paxbx-whf5g.png)

Reply:
I did some scanning across the theme and we're pretty consistently naming this the block type. Also it requires you to navigate to `*.schema.json` to know where this is coming from.